_opensearch/query-dsl/query-string.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Query string queries
+parent: Query DSL
+nav_order: 70
+---
+
+# Query string queries
+
+A `query_string` query parses the query string based on the `query_string` syntax. 
+
+## Example
+
+The following query searches for the speaker `KING` in the play name that ends with `well`:
+
+```json
+GET shakespeare/_search
+{
+ "query": {
+    "query_string": {
+      "query": "speaker:KING  AND play_name: *well"
+    }
+  }
+}
+```
+
+## Parameters
+
+Query string query supports the following parameters. All parameters except `query` are optional.
```

Comment:
Should this follow the same format as the first sentence, that is, "`query string` query?"